source/patterns/@aws-solutions-constructs/aws-route53-apigateway/README.md

```diff
@@ -0,0 +1,102 @@
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_route53_apigateway`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-route53-apigateway`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.route53apigateway`|
+
+## Overview
+
+This AWS Solutions Construct implements an Amazon Route 53 connected to a configured Amazon API Gateway REST API.
+
+Here is a minimal deployable pattern definition in Typescript:
+
+``` typescript
+import * as api from '@aws-cdk/aws-apigateway';
+import * as lambda from "@aws-cdk/aws-lambda";
+import * as route53 from "@aws-cdk/aws-route53";
+import { Route53ToApigateway } from '@aws-solutions-constructs/aws-route53-apigateway';
+
+// The construct requires an existing REST API, this can be created in raw CDK or extracted
+// from a previously instantiated construct that created an API Gateway REST API
+const existingRestApi = previouslyCreatedApigatewayToLambdaConstruct.apiGateway;
+
+const ourHostedZone = route53.HostedZone.fromLookup(this, 'HostedZone', {
+    domainName: "example.com",
+  });
+
+// This construct can only be attached to a configured API Gateway.
+new Route53ToApigateway(this, 'Route53ToApigatewayPattern', {
+    existingApiGatewayObj: existingRestApi,
+    existingHostedZoneInterface: ourHostedZone,
+    publicApi: true
+});
+
+```
+
+## Initializer
+
+``` text
+new Route53ToApigateway(scope: Construct, id: string, props: Route53ToApigatewayProps);
+```
+
+_Parameters_
+
+* scope [`Construct`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_core.Construct.html)
+* id `string`
+* props [`Route53ToApigatewayProps`](#pattern-construct-props)
+
+## Pattern Construct Props
+
+This construct cannot create a new Public Hosted Zone, if you are creating a public API you must supply an existing Public Hosted Zone that will be reconfigured with a new Alias record. Public Hosted Zones are configured with public domain names and are not well suited to be launched and torn down dynamically, so this construct will only reconfigure existing Public Hosted Zones.
+
+This construct can create Private Hosted Zones. If you want a Private Hosted Zone, then you can either provide an existing Private Hosted Zone or a privateHostedZoneProps value with at least the Domain Name defined. A private API will already define the VPC and we only accept existing APIs, so this construct will never create a VPC. Users can bring private APIs they created via the CDK through the existingVpc prop.
```

Comment:
This was good info to include. A few edits:

This construct can create Private Hosted Zones. If you want a Private Hosted Zone, then you can either provide an existing Private Hosted Zone or a privateHostedZoneProps value with at least the Domain Name defined. A private Rest API already exists in a VPC, so that VPC must be provided in the existingVpc prop. There is no scenario where this construct can create a new VPC (since it can't create a new API), so the vpcProps property is not supported on this construct.